src/services/dashboards/resource.js

```diff
@@ -418,7 +418,23 @@ async function GenerateFlatTempTables(reportIds, tblNames) {
       JOIN ${tblNames.createdResourcesTempTableName} arorr
         ON aror."resourceId" = arorr.id;
 
-      -- 7.) Create date headers.
+      -- 7.) Create flat reports with courses temp table.
```

Comment:
We don't seem to have any tests for this SQL as we do for the rest of this service? Sorry if I'm missing them